src/style.css

```diff
@@ -1,3 +1,77 @@
 body {
+  background-color: #ddd;
+  color: #000;
   margin: 0;
+  min-width: 1024px;
+  font-size: 12px;
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  line-height: normal;
+  text-transform: uppercase;
+  --blue-color: #00acdc;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: #fff;
+  padding: 0 50px;
+}
+
+.img__link {
+  display: flex;
+  margin: 0;
+  text-decoration: none;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+:not(:first-of-type).nav__item {
+  margin-left: 20px;
+}
+
+.nav__link {
+  display: flex;
+  align-items: center;
+  text-decoration: none;
+  height: 40px;
+  padding: 10px 0;
+  position: relative;
+  color: var(--blue-color);
+}
+
+:not(:first-of-type) .nav__link:link,
+:not(:first-of-type) .nav__link:visited {
+  color: #000;
+}
+
+:not(:first-of-type) .nav__link:hover {
+  color: var(--blue-color);
+}
+
+.nav__link:after {
+  display: none;
+}
+
+.is-active:after,
+.nav__link:after {
+  content: "";
+  position: absolute;
+  width: 100%;
+  height: 4px;
+  bottom: 0;
+  background: var(--blue-color);
+  border-radius: 2px;
+}
+
+.is-active:after,
+.nav__link:active:after {
+  display: block;
 }
```

Comment:
Oh, it can be done more easily. Let me give you a hint :)
By default, our color is black, but where the `is-active` class is mixed, it will be changed to blue. Also, the `is-active` class will have a `::after` pseudo-element that will draw a blue rectangle. And the hover effect is simply applied to the `.nav__link'

Reply:
upd demo https://natamrshn.github.io/layout_moyo-header/